.github/workflows/conda.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         platform: ['ubuntu-latest','windows-latest','macos-latest']
 
     env:
-      PLATFORM: ${{ matrix.platform }}
```

Comment:
I was just curious if the name `PLATFORM` was actually messing with anything. 👍 on the name change though.

Reply:
vcvarsall.bat sets that variable (using a different case arrangement), which was the root of our problem.

Reply:
ah. brutal.